README.md

```diff
@@ -1,4 +1,46 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) *Completed*
-2. [Core Java Apps](./core_java) *Completed*
-3. [Springboot Trading REST API](./springboot) *In-progress*
+# Steven Liu . Jarvis Consulting
+
+Problem solver, culture explorer, lifelong learner. I'm a Jarvis data engineer associate graduated from the University of Saskatchewan with an Honors B.Sc. degree in Computer Science and Statistics. The blend of academic training has given me the knowledge of both solid coding and analytical background. I'm ready to kickstart my career in data engineering, which includes all the exciting things I am passionate about. I look forward to opportunities to utilize and develop my skills from Jarvis' clients in Toronto.
```

Comment:
Please run a spelling and grammar check using https://www.grammarly.com/ if necessary